<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaiseRequestDialog</class>
 <widget class="QDialog" name="RaiseRequestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Backlog Item</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Add New Backlog Item</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="typeLabel">
       <property name="text">
        <string>Request Type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="typeComboBox">
       <item>
        <property name="text">
         <string>Backlog Item</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change Request</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bug Report</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Feature</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Epic</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="parentLabel">
       <property name="text">
        <string>Parent:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="parentComboBox"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QTextEdit" name="descriptionTextEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="priorityLabel">
       <property name="text">
        <string>Priority:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="priorityComboBox">
       <item>
        <property name="text">
         <string>Low</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>High</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="complexityLabel">
       <property name="text">
        <string>Complexity:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="complexityComboBox">
       <item>
        <property name="text">
         <string>Small</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Medium</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Large</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>